.clang-tidy

```diff
@@ -37,8 +37,7 @@ Checks: "*,
 
         google-readability-avoid-underscore-in-googletest-name,
         google-upgrade-googletest-case,
--*,
-readability-braces-around-statements,
```

Comment:
Did you mean to delete this line as well? (readability-braces-around-statements)

Reply:
It's a bugfix from #332